### DISCUSSION TITLE
How do you undo unpowering a device?

### DISCUSSION THREAD

**Discussion by tyilo** [coldfix/udiskie#257] [Q&A]
Specifically I'm interested in my SD card reader.

Is the only option to reboot my system or can I run some command to power it on again?

**Comment by coldfix** [comment]
Hi,

there is nothing in udisks as far as I know, but you could try [usbreset](https://wiki.ubuntuusers.de/usbreset/) or try [rescanning](https://unix.stackexchange.com/questions/73908/how-to-reset-cycle-power-to-a-pcie-device).

(and of course unplug/replug the device, but I assume you were excluding that option intentionally too)
